tests/subsys/mgmt/mcumgr/all_options/prj.conf

```diff
@@ -4,8 +4,6 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 CONFIG_ZTEST=y
-CONFIG_TINYCRYPT=y
-CONFIG_TINYCRYPT_SHA256=y
```

Comment:
this needs to have mbedtls enabled, as the name implies this test ensures all options can be enabled